src/plugins/intel_gna/docs/transformations.md

```diff
@@ -0,0 +1,97 @@
+# GNA transformations documentation
+
+GNAPlugin has multiple methods, one of the most important is LoadNetwork. That method prepares input graph to execute on the GNA hardware. Input graph usually consists of variety of operations, i.e. Convolution, Add, Gather, LSTMSequence. GNA hardware provides limited number of operations. First of all LoadNetwork method converts input network to equivalent network with layers each of them could be executed on the GNA hardware.
+There is a work moving entire GNA plugin from legacy CNNNetwork API to the ngraph.
+
+## LoadNetwork
+
+GNAPlugin::LoadNetwork will start with the next stages:
```

Comment:
```
GNAPlugin::LoadNetwork will start with the next stages:
-	Converting input graph to fully GNA-supported graph (all in ngraph)
-	Creating and connecting GNA primitives within libGNA from ngraph-based network
```
If this chapter means how LoadNetwork  should work in the future:
 I would change `GNAPlugin::LoadNetwork will start with the next stages:` to
GNAPlugin::LoadNetwork in the future should execute following stages:
-	Converting input graph to fully GNA-supported graph (all in ngraph)
-	Creating and connecting GNA primitives within libGNA from ngraph-based network